governance-and-automation/optimize-storage.md

```diff
@@ -75,12 +75,44 @@ Once you detect the largest sites with the help of the Storage Metrics report, u
 
 ![Storage Metrics Report in Point](../.gitbook/assets/optimize-storage-storage-metrics-report-point.png)
 
-There are also the [Site Storage](../../point-collaborators/resolve-governance-tasks/tenant-storage-limit.md#site-storage) and the [File Storage](../../point-collaborators/resolve-governance-tasks/tenant-storage-limit.md#file-storage) reports available. 
+From the Storage Metrics Report, you can also access the [Site Storage](../../docs-point/reporting/cleanup-and-health-reports.md#site-storage-metrics) and the [File Storage](../../docs-point/reporting/cleanup-and-health-reports.md#file-storage-details) reports.
+
+There, you can gain deeper **insight into how your storage is used by specific file types**, such as videos, photos, office documents, and others. 
+
+You can also see how much **storage you can save per site** if you delete file versions older than a certain number of months or every file version except for the most recent one. 
+
+It is also where you can **complete the Clean Up File Versions action and free up space**. 
+
 
 ## Free Up Storage
 
-After you click the **link in the Storage Metrics URL column**, the SharePoint site-specific Storage Metrics report opens in your browser.
-Here, you can find a detailed view of content size in your SharePoint site and the percentage of storage it uses.
+There are two ways you can free up storage for your tenant. One can be completed from within Syskit Point, while the other requires you to remove it directly from the SharePoint site. Below, you can find instructions on both options. 
```

Comment:
isto bi stavio bulete
There are two ways you can free up storage, by using:
* Syskit Point + link/anchor na sekciju ###Syskit Point
* SharePoint + link/anchor na sekciju ### SharePoint 
ispod odvojiti tekst u dva podnaslova
###Syskit Point
### SharePoint